testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/multinode-postgresql-spooling/coordinator-config.properties

```diff
@@ -16,3 +16,7 @@ task.max-writer-count=2
 # Enable spooling protocol
 experimental.protocol.spooling.enabled=true
 protocol.spooling.shared-secret-key=jxTKysfCBuMZtFqUf8UJDQ1w9ez8rynEJsJqgJf66u0=
+# Enable direct storage access so we can test fetching against the storage
```

Comment:
`Switch spooled product tests to direct location` would it be too costly to test both direct and indirect?

Reply:
No, but this would require a separate env with a separate config

Reply:
Indirect is tested in the DistributedQueryRunner

Reply:
Maybe add the comment that there is coverage so it is obvious